web/res/js/membershipForm.js

```diff
@@ -162,7 +162,8 @@ $( function () {
 						'BEZ': 'Lastschrift',
 						'UEB': 'Überweisung',
 						'MCP': 'Kreditkarte',
-						'PPL': 'PayPal'
+						'PPL': 'PayPal',
+						'SUB': 'Sofortüberweisung'
```

Comment:
Hmm, this option object seems to be a copy-paste relict from `donationForm`. The only payment option for memberships is `BEZ`, with `PPL` in planning. As long as Sofortüberweisung does not support standing orders (dauerauftrag), it will never be an option for memberships, same thing with MCP. I'm torn between removing unused options for clarity and leaving them in for "future-proofing" and consistency. You decide.